waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -105,6 +105,17 @@
          (clojure.data/diff expected# actual#)))
      (is (= expected# actual#))))
 
+(deftest replicaset-spec-no-image
+  (let [scheduler (make-dummy-scheduler ["test-service-id"])
+        replicaset-spec ((:replicaset-spec-builder-fn scheduler) scheduler "test-service-id" dummy-service-description)]
+    (is (= "twosigma/kitchen:latest" (get-in replicaset-spec [:spec :template :spec :containers 0 :image])))))
```

Comment:
This test will fail if a different configuration file is used. We should try reading this value from the settings instead.

We might need a new helper function to pull this value, since it's in a different place depending on whether or not we're running a composite scheduler.

Maybe use this function as a starting point:
https://github.com/twosigma/waiter/blob/d09cb7cbd3942da7b541b936873af9834eb2264c/waiter/src/waiter/util/client_tools.clj#L857-L863